state/stateConsensus.go

```diff
@@ -2389,15 +2388,16 @@ func (s *State) ProcessDBSig(dbheight uint32, msg interfaces.IMsg) bool {
 			s.LogPrintf("dbsig", "1st ProcessDBSig(): %10s DBSig dbht %d leaderheight %d VMIndex %d Timestamp %x %d, leadertimestamp = %x %d",
 				s.FactomNodeName, dbs.DBHeight, s.LLeaderHeight, dbs.VMIndex, dbs.GetTimestamp().GetTimeMilli(), dbs.GetTimestamp().GetTimeMilli(), s.LeaderTimestamp.GetTimeMilliUInt64(), s.LeaderTimestamp.GetTimeMilliUInt64())
 
-			cbtx := fs.GetCurrentBlock().(*factoid.FBlock).Transactions[0].(*factoid.Transaction)
-			foo := cbtx.MilliTimestamp
+			cb := fs.GetCurrentBlock()
+			cbtx := cb.GetTransactions()[0]
+			foo := cbtx.GetTimestamp().GetTimeMilliUInt64()
```

Comment:
is this variable really named foo..... I know it's not you originally but from now on git will attribute it to you, so you may want to rename :P

Reply:
> is this variable really named foo..... I know it's not you originally but from now on git will attribute it to you, so you may want to rename :P

Gave them all more appropriate names.